includes/admin/class-wc-payments-admin.php

```diff
@@ -780,7 +780,7 @@ private function get_js_settings(): array {
 				'isWelcomeTourDismissed' => WC_Payments_Features::is_fraud_protection_welcome_tour_dismissed(),
 			],
 			'accountDefaultCurrency'      => $this->account->get_account_default_currency(),
-			'frtDiscoverBannerSettings'   => get_option( 'wcpay_frt_discover_banner_settings', '' ),
+			'frtDiscoverBannerDismissed'  => get_option( 'wcpay_frt_discover_banner_dismissed' ),
```

Comment:
I just realized that this change will make the discoverability banner visible for all merchants again. 😱

Maybe we should roll back to the last commit before the naming change?

Reply:
That was a good catch! I'll re-work it tomorrow morning.